include/zephyr/mgmt/mcumgr/mgmt/mgmt.h

```diff
@@ -174,14 +174,15 @@ typedef void (*mgmt_reset_buf_fn)(void *buf, void *arg);
  *
  * @return 0 if a response was successfully encoded, #mcumgr_err_t code on failure.
  */
-typedef int (*mgmt_handler_fn)(struct smp_streamer *ctxt);
+typedef int (*mgmt_handler_fn)(struct smp_streamer *ctxt, void *user_data);
```

Comment:
Add Doxygen comment for the new parameter

Reply:
Thank you! Will update the PR if the RFC is approved :)